pkg/collect/certificates_test.go

```diff
@@ -119,109 +123,196 @@ po/R2TjyxywLn4DgM7BAzzu9qfHWf+S4eQjRUHQshPbUEX9CEsSd5tCu8ZHVbBds
+			Collectors: []troubleshootv1beta2.Collect{
+				{
+					Certificates: &troubleshootv1beta2.Certificates{
+						CollectorMeta: troubleshootv1beta2.CollectorMeta{
+							CollectorName: "collectorname",
+						},
+						Name: "expired certificate",
+						Secrets: []troubleshootv1beta2.CertificateSource{
+							{
+								Name:       "expiredCert",
+								Namespaces: []string{"test"},
+							},
+						},
+					},
+				},
+			},
+			want: []CertCollection{
+				{
+					Source: &CertificateSource{
+						Namespace:  "test",
+						SecretName: "expiredCert",
+					},
+					CertificateChain: []ParsedCertificate{
+						{
+							CertName:                "tls.crt",
+							Subject:                 "O=Internet Widgits Pty Ltd,ST=Some-State,C=AU",
+							SubjectAlternativeNames: []string(nil),
+							Issuer:                  "O=Internet Widgits Pty Ltd,ST=Some-State,C=AU",
+							NotAfter:                time.Date(2015, time.September, 12, 21, 52, 2, 0, time.UTC),
+							NotBefore:               time.Date(2012, time.September, 12, 21, 52, 2, 0, time.UTC),
+							IsValid:                 false,
+							IsCA:                    true,
+						},
+					},
+				},
 			},
 		},
 		{
-			name:        "multi certificate",
-			certName:    "digicert",
-			certificate: []byte(multiCertChain),
-			expiredTime: time.Date(2024, 1, 1, 0, 0, 0, 0, time.UTC),
-			want: parsedCertResult{
-				CertQty:   2,
-				IsSubject: true,
-				IsValid:   []bool{false, true},
+			name:          "multiple certificate",
+			certChainName: "multiCert",
+			Collectors: []troubleshootv1beta2.Collect{
+				{
+					Certificates: &troubleshootv1beta2.Certificates{
+						CollectorMeta: troubleshootv1beta2.CollectorMeta{
+							CollectorName: "collectorname",
+						},
+						Name: "multiple certificate",
+						Secrets: []troubleshootv1beta2.CertificateSource{
+							{
+								Name:       "multiCert",
+								Namespaces: []string{"test"},
+							},
+						},
+					},
+				},
+			},
+			want: []CertCollection{
+				{
+					Source: &CertificateSource{
+						Namespace:  "test",
+						SecretName: "multiCert",
+					},
+					CertificateChain: []ParsedCertificate{
+						{
+							CertName:                "tls.crt",
+							Subject:                 "CN=DigiCert High Assurance EV CA-1,OU=www.digicert.com,O=DigiCert Inc,C=US",
+							SubjectAlternativeNames: []string(nil), Issuer: "CN=DigiCert High Assurance EV Root CA,OU=www.digicert.com,O=DigiCert Inc,C=US",
+							NotAfter:  time.Date(2021, time.November, 10, 0, 0, 0, 0, time.UTC),
+							NotBefore: time.Date(2007, time.November, 9, 12, 0, 0, 0, time.UTC),
+							IsValid:   false,
+							IsCA:      true,
+						},
+						{
+							CertName:                "tls.crt",
+							Subject:                 "CN=DigiCert High Assurance EV Root CA,OU=www.digicert.com,O=DigiCert Inc,C=US",
+							SubjectAlternativeNames: []string(nil), Issuer: "CN=DigiCert High Assurance EV Root CA,OU=www.digicert.com,O=DigiCert Inc,C=US",
```

Comment:
nit: This can just be `nil` and add new line before `Issuer`. `make test` will format the code

Reply:
Thank you! formatted all the tests code